neptune/new/envs.py

```diff
@@ -36,3 +36,5 @@
 NEPTUNE_SYNC_BATCH_TIMEOUT_ENV = "NEPTUNE_SYNC_BATCH_TIMEOUT"
 
 NEPTUNE_SUBPROCESS_KILL_TIMEOUT = "NEPTUNE_SUBPROCESS_KILL_TIMEOUT"
+
+NEPTUNE_GET_ALL_ITEMS_STEP_SIZE = "NEPTUNE_GET_ALL_ITEMS_STEP_SIZE"
```

Comment:
How about `NEPTUNE_FETCH_TABLE_STEP_SIZE`?

Reply:
no problem, I will change that